gazelle/python/python_test.go

```diff
@@ -42,9 +41,10 @@ const (
 	gazelleBinaryName = "gazelle_binary"
 )
 
-var gazellePath = mustFindGazelle()
+var gazellePath string
```

Comment:
Why is this necessary? Since this is a global variable, it may be better to set it in `TestMain` or as it was done previously?

Reply:
`var gazellePath = mustFindGazelle()` will make `go test .` failed. Although this will not affect bazel test.

I simply modified it, and no longer use global variables. There is no need for global variables here.